crates/iroha_multisig_data_model/src/lib.rs

```diff
@@ -36,39 +51,62 @@ type Weight = u8;
 /// Default multisig transaction time-to-live in milliseconds based on block timestamps
 pub const DEFAULT_MULTISIG_TTL_MS: u64 = 60 * 60 * 1_000; // 1 hour
 
-/// Arguments to propose or approve multisig transaction
-#[derive(Debug, Clone, Decode, Encode, Serialize, Deserialize, IntoSchema)]
-pub enum MultisigTransactionArgs {
-    /// Propose instructions and initialize approvals with the proposer's one
-    Propose(Vec<InstructionBox>),
-    /// Approve certain instructions
-    Approve(HashOf<Vec<InstructionBox>>),
+/// Propose a multisig transaction and initialize approvals with the proposer's one
+#[derive(Debug, Clone, Encode, Decode, Serialize, Deserialize, IntoSchema, Constructor)]
+pub struct MultisigPropose {
+    /// Multisig account to propose
+    pub account: AccountId,
+    /// Proposal contents
+    pub instructions: Vec<InstructionBox>,
```

Comment:
I think you should verify on decode/deserialize that it's not empty. Or do we allow empty?